### ISSUE TITLE
Changed Encoding Handling since v2.8.8  Breaks Nuclei-Templates

### ISSUE THREAD

**Issue by Lednerb** [projectdiscovery/nuclei#3329]
### Nuclei version:
\> v2.8.7

### Current Behavior:
In version 2.8.7 the [CVE-2022-28923](https://github.com/projectdiscovery/nuclei-templates/blob/main/cves/2022/CVE-2022-28923.yaml) template works fine and sends the required request as-is.
Since version 2.8.8 the vulnerability can no longer be detected, because the request is changed when sending to the vulnerable server.

### Expected Behavior:
Requests should be send in the same format as they are defined in the templates.

### Steps To Reproduce:
1. Start the vulnerable caddy server
2. Start Burp with default settings
3. Download the [CVE-2022-28923](https://github.com/projectdiscovery/nuclei-templates/blob/main/cves/2022/CVE-2022-28923.yaml) template to your current directory
4. Run `./nuclei-2.8.7 -t . -u http://localhost:3333 -H 'NucleiVersion:2.8.7' -p http://127.0.0.1:8080`
5. Run `./nuclei-2.8.8 -t . -u http://localhost:3333 -H 'NucleiVersion:2.8.8' -p http://127.0.0.1:8080`
6. nuclei version 2.8.7 detects the vulnerability, v.2.8.8 not.

![Pasted image 20230216172928](https://user-images.githubusercontent.com/2056876/219432440-736f6026-0dd8-423a-a20d-c8097179c9e8.png)

Attached you also find the burp file with the above mentioned requests.
[2023-02-16-Nuclei-URL-Issue.zip](https://github.com/projectdiscovery/nuclei/files/10758398/2023-02-16-Nuclei-URL-Issue.zip)

### TL;DR
v2.8.7 sends the correct request:
```http
GET /%5C%5Cinteract.sh/%252e%252e%252f HTTP/1.1
```

v2.8.8 and above sens the false request:
```http
GET /%255C%255Cinteract.sh/%25252e%25252e%25252f HTTP/1.1
```

**Comment by ehsandeep** [comment]
@Lednerb it was a regression issue fixed in follow-up release, i.e [v2.8.9](https://github.com/projectdiscovery/nuclei/releases/tag/v2.8.9), do you experience the same issue with latest version as well?

**Comment by ehsandeep** [comment]
@tarunKoyalwar another related issue - https://github.com/projectdiscovery/nuclei/issues/3334
